JavaIsland/i18n-auto.json

```diff
@@ -3511,19 +3511,7 @@
 }, {
   "id" : "SeekForHighPayingJob",
   "data" : {
-    "EN" : "Seek for High-paying Job?",
-    "ZH_HANS" : "在寻找高薪工作？",
-    "ZH_HANT" : "在尋找高薪工作？",
-    "ES" : "¿Busca un trabajo bien remunerado?",
-    "AR" : "هل تبحث عن وظيفة ذات رواتب عالية؟",
-    "PT" : "Procura um emprego bem remunerado?",
-    "ID" : "Mencari Pekerjaan Bergaji Tinggi?",
-    "FR" : "À la recherche d'un emploi bien rémunéré ?",
-    "JA" : "高給の仕事をお探しですか？",
-    "RU" : "Ищете высокооплачиваемую работу?",
-    "DE" : "Auf der Suche nach einem hochbezahlten Job?",
-    "KO" : "고임금 직업을 찾으십니까?",
-    "IT" : "Cerchi un lavoro ben pagato?"
+    "ZH_HANS" : "在寻找高薪工作？"
```

Comment:
As I said in another PR, the JSONs should not be modified. But never mind, the automation tool will overwrite these changes when building so I'll just merge as it is.